src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
```

Comment:
Please also add a docblock.  And also, why did you need this as a static class?  It's already instantiated and this method can't be used outside of this class.